.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+# What it does
+
+> Describe here the changes included in the PR
+
```

Comment:
I suggest to add `Design consideration`

Reply:
What do you mean by it? What should be the content for that?